modules/ROOT/pages/about-mule-message.adoc

```diff
@@ -150,6 +147,44 @@ Notice that each attribute is a key/value pair (such as `fileName=myJson.json`).
 * `attributes.'fileName'` to return the file name: `myJson.json`.
 * `attributes.size` to return the size of the file: `22`.
 
+=== Valid Identifiers for Attribute Names
+
+Attribute names that follow the rules described in xref:dataweave-language-introduction.adoc#valid-identifiers[Rules for Declaring Valid Identifiers] can be accessed and declared without using any quotation marks.
+
+To use an attribute name that is not a valid identifier, surround the attribute name with any of the following characters:
+
+* single quotation marks (`'`)
+* double quotation marks (`"`)
+* backticks (```)
+
+For example, consider the following DataWeave variable declaration:
+
+[source,java,linenums]
+----
+var myVar = {
+              id : "1234",
+              "123 abc !@#" : "some_value"
+            }
+----
+
+Because attribute name `123 abc !@#` is not a valid identifier, you must use quotation marks or backticks to declare it.
+
+To access the value from attribute `123 abc !@#`, which is `"some_value"`, use quotation marks or backticks. For example:
+
+[source]
+----
+myVar.'123 abc !@#'
+----
+
+It is not required to use quotation marks or backticks to declare an attribute or to select a value from an attribute when the attribute name is a valid identifier, but it can be done.
```

Comment:
"but it can be done" : I believe Kim will ask for active voice where possible, e.g., "but you can use these characters around them" or something similar

Reply:
Duke's right!
```suggestion
Using quotation marks or backticks to declare an attribute or select a value from an attribute when the attribute name is a valid identifier is optional.
```